### PR TITLE
feat: update-ui contracts list

### PR DESCRIPTION
**Pull request by qiweiii**
new ui for contracts list page: `/contracts`

### CODE REVIEW

**Comment by qiweiii** [comment]
@Kirl70 Seems we don't have this button for contract list:

image

**Comment by Kirl70** [comment]
> @Kirl70 Seems we don't have this button for contract list:
> 
> image

it's the original this button:

image

**Comment by qiweiii** [comment]
@Kirl70 Do we have this in contracts page? the page your are showing is tokens page

**Comment by Kirl70** [comment]
> @Kirl70我们在合同页面中有这个吗？您显示的页面是令牌页面



> @Kirl70 Do we have this in contracts page? the page your are showing is tokens page

Sorry i made a mistake, it has been deleted

**Comment by FrederLu** [comment]
image
image

@Kirl70 
Do these 2 copy/style need to refer to UI modification?

**Comment by Keith-CY** [comment]
> image
> 
> image
> 
> @Kirl70 Do these 2 copy/style need to refer to UI modification?

The `CKB` of point 1 has been updated to `pCKB` when godwoken v1 launched

**Comment by Kirl70** [comment]
The text information is changed according to the needs and does not need to be adjusted according to the design draft, but the information of Txn needs to be right-aligned

image

File: components/TableCell.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { TableCell, tableCellClasses } from '@mui/material'
```
Comment (Keith-CY) [inline components/TableCell.tsx:1]: `components/Table` could be reused

Comment (Keith-CY) [reply components/TableCell.tsx:1]: as https://github.com/Magickbase/godwoken-explorer-ui/blob/update-ui/components/TxList/index.tsx#L164-L249

**Comment by qiweiii** [comment]
Fixed.

There is something wrong with existing table header 'Roboto' font wieight, will be fixed together in this https://github.com/Magickbase/godwoken_explorer/issues/789

**Comment by FrederLu** [comment]
https://github.com/Magickbase/godwoken-explorer-ui/pull/360#issuecomment-1196407707
https://github.com/Magickbase/godwoken-explorer-ui/pull/360#issuecomment-1196395864

Verified.

**Review by FrederLu — APPROVED** [review]

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**